src/index.js

```diff
@@ -0,0 +1,199 @@
+const debug = require('debug')('dynamodb')
```

Comment:
can we make debug have `raziel` in it? I wouldnt want to cause possible confusion. maybe `raziel-dynamodb`